chapters/ch10/1.setup-the-movie-db.ipynb

```diff
@@ -10,13 +10,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "sys.path.append('../..')\n",
-    "from aips import get_engine, get_ltr_engine\n",
+    "from aips import get_engine, get_ltr_engine, set_engine\n",
```

Comment:
Is `set_engine` used? If not, it can be removed from imports.

Reply:
Removed